chia/timelord/timelord_launcher.py

```diff
@@ -90,27 +114,33 @@ async def spawn_all_processes(config: Dict, net_config: Dict, lock: asyncio.Lock
         log.info("Process_count set to 0, stopping TLauncher.")
         return
     awaitables = [
-        spawn_process(hostname, port, i, lock, prefer_ipv6=net_config.get("prefer_ipv6", False))
+        spawn_process(
+            hostname,
+            port,
+            i,
+            process_mgr,
+            prefer_ipv6=net_config.get("prefer_ipv6", False),
+        )
         for i in range(process_count)
     ]
     await asyncio.gather(*awaitables)
 
 
 async def async_main(config: Dict[str, Any], net_config: Dict[str, Any]) -> None:
```

Comment:
Given that we're changing runtime timelord stuff, let's create an ephemeral testnet for this when we get close to the end.  We can leave this conversation open until we take care of that.